src/ScrollSync.jsx

```diff
@@ -129,7 +129,10 @@ export default class ScrollSync extends Component {
           this.syncScrollPosition(scrolledPane, pane)
           /* Re-attach event listeners after we're done scrolling */
           window.requestAnimationFrame(() => {
-            this.addEvents(pane, groups)
```

Comment:
The cause of the problem is that `pane` would belong to multiple groups, and the event listeners would only be reset to some of these groups here.

In reference to the GIF in the linked issue, the pane in pink belongs to `["horizontal", "vertical"]`. But when scrolling vertically on the blue grid, we would reset the event listeners of the pink grid to `"vertical"` and therefore break the horizontal scrolling synchronization between the pink and black grids.